.github/workflows/test-run-cypress-tests-action.yml

```diff
@@ -11,20 +11,11 @@ jobs:
     runs-on: ubuntu-latest-4-cores
 
     steps:
+      # We need to checkout code in order to run a local action
       - name: Checkout code
```

Comment:
Shouldn't this "Checkout code" step be removed because it is part of `run-ci`?

An old version of the action is used by the way.